test/integrations/component.test.ts

```diff
@@ -50,7 +53,10 @@ if (opts.generate === 'true') {
 
 let server: Server;
 
+const REPORT_COMPATIBLE_INTEGRATION = ['klaviyo'];
```

Comment:
Can't we move this to the destination test file itself? Otherwise, with every destination update we need to update this array.

Reply:
Since there are multiple folders `processor`, router`, `deletion` I am not sure which file to keep this info in. I can create a new file under destination directory to keep this info but it seems like an overkill. 

Till we migrate all destinations to this structure we need to maintain a list hence I thought if we keep in a central place it would be better. If we are ok with creating a new metadata file under test directory for these destinations we can go with that